app/api/utils.py

```diff
@@ -455,6 +455,39 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                              allow_nan=not self.strict) + '\n'
 
 
+class FastTextPainter(object):
+
+    @staticmethod
+    def paint_labels(documents, labels):
+        serializer = DocumentSerializer(documents, many=True)
+        serializer_labels = LabelSerializer(labels, many=True)
+        data = []
+        for d in serializer.data:
+            labels = []
+            for a in d['annotations']:
+                label_obj = [x for x in serializer_labels.data if x['id'] == a['label']][0]
+                labels.append('__label__{}'.format(label_obj['text'].replace(' ', '_')))
+            data.append('{} {}'.format(' '.join(labels), d['text'].replace('\n', ' ')) if labels else d['text'].replace('\n', ' '))
+        return data
+
+
+class PlainTextRenderer(BaseRenderer):
+    media_type = 'text/plain'
+    format = 'txt'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        if data is None:
+            return bytes()
+
+        if not isinstance(data, list):
+            data = [data]
+
+        buffer = io.BytesIO()
+        for d in data:
+            buffer.write((d + '\n').encode(self.charset))
```

Comment:
set `self.charset` to utf-8 explicitly. Currently it's relying on the definition in `BaseRenderer`.